web/docs/tql2/functions/decode_hex.md

```diff
@@ -0,0 +1,32 @@
+# decode_hex
+
+Decodes bytes from their hexadecimal representation.
+
+```tql
+decode_hex(bytes: blob|string) -> string
+```
+
+## Description
+
+Decodes bytes from their hexadecimal representation.
+
+### `bytes: blob|string`
+
+The value to decode.
+
+## Examples
+
+### Decode a blob from hex
+
+```tql
+from {bytes: "54656E7A6972"}
+decoded = bytes.decode_hex()
+```
+
+```tql
+{bytes: "54656E7A6972", decoded: "Tenzir"}
+```
```

Comment:
Can you also decode a lower-case hex string? We should support this, and an extra example would be helpful.

Reply:
It already decodes upper, lower and mixed-case strings.

Reply:
Added a simple example of decoding mixed-case, and a couple simple tests.